src/components/apis/list-of-apis/react/runtime/ApiListRuntime.tsx

```diff
@@ -12,6 +12,7 @@ import { FiltersPosition } from "../../listOfApisContract";
 import { TApisData } from "./utils";
 import { ApiListTableCards } from "./ApiListTableCards";
 import { ApiListDropdown } from "./ApiListDropdown";
+import { Tag } from "../../../../../models/tag";
```

Comment:
Any reason we can't add alias for src root so this import will be `@/models/tag`?

Reply:
Good improvement but not related to this fix.
Can be implemented later in a dedicated PR